### PR TITLE
#81 + #4

### PR DESCRIPTION
**Pull request by alisonailea**
Grids: Normalizing Flexbox for cross-browsers
These fixes are only needed until IE12 becomes standard. I also introduced a Normalize mixin that can be used in other components.
Also adding the checklist since it wasn't included in the original Grids PR
- [x] Manually tested across supported browsers
 - [x]  Chrome
 - [x]  Firefox
 - [x]  Safari
 - [x]  IE11
- [x]  Unit tests written (common at minimum)
- [x]  PR has one of the semver- labels
- [ ]  Two core team engineer approvals
- [ ]  One core team UX approval

### CODE REVIEW

File: src/styles/mixins.less [hunk 1]
```diff
@@ -90,3 +90,457 @@
 	-webkit-transform:translate3d(0,0,0);
 }
 
+
```
Comment (ogupte) [inline src/styles/mixins.less]: this code should exist as a separate source file and be included

File: src/docs/index.less [hunk 2]
```diff
@@ -4,6 +4,11 @@
 
 @import 'http://yui.yahooapis.com/pure/0.6.0/pure-min.css';
 @import '../styles/variables';
+@import '../styles/mixins';
+
+// Normalize
+#Normalize > .html;
```
Comment (jondlm) [inline src/docs/index.less]: Are these required for the grid to behave correctly?

File: src/styles/mixins.less [hunk 3]
```diff
@@ -90,3 +90,457 @@
...
+/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+#Normalize {
```
Comment (jondlm) [inline src/styles/mixins.less]: I'd suggest we keep this lowercase `#normalize` so it doesn't make people think Normalize is a react component (our existing convention).

File: src/styles/mixins.less (hunk 1 above)
Comment (jondlm) [inline src/styles/mixins.less]: Agreed, perhaps `mixins-normalize.less`

File: src/styles/mixins.less (hunk 3 above)
Comment (nathaniellee) [inline src/styles/mixins.less]: +1

File: src/docs/index.less (hunk 2 above)
Comment (alisonailea) [inline src/docs/index.less]: No but I wanted to normalize our demo page

File: src/styles/mixins.less (hunk 3 above)
Comment (alisonailea) [inline src/styles/mixins.less]: That makes sense. But I do want to differentiate a Mixin-group from a basic mixin

File: src/styles/mixins/normalize.less [hunk 4]
```diff
@@ -0,0 +1,453 @@
+/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+#mixin-Normalize {
```
Comment (jondlm) [inline src/styles/mixins/normalize.less]: Mixin is already implied by the use of a less namespace, also, could you make this non capitalized? It conflicts with an existing convention.

File: src/styles/mixins/normalize.less [hunk 5]
```diff
@@ -0,0 +1,453 @@
+/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+#normalize {
```
Comment (alisonailea) [inline src/styles/mixins/normalize.less]: switched name as per @jondlm suggestion

File: .nvmrc [hunk 6]
```diff
@@ -0,0 +1 @@
+v4.2
```
Comment (alisonailea) [inline .nvmrc]: https://github.com/creationix/nvm

**Pull request merged**